sass/layout/hero.sass

```diff
@@ -142,6 +142,14 @@
       .hero-body
         padding-bottom: 18rem
         padding-top: 18rem
+  &.is-halfheight
+    min-height: 50vh
+    .hero-body
+      align-items: center
+      display: flex
+      & > .container
+        flex-grow: 1
+        flex-shrink: 1
```

Comment:
There's a lot of duplication. You can regroup most of `is-halfheight` and `is-fullheight`.